README.md

```diff
@@ -8,8 +8,8 @@ Sometimes your rows might come in a nested format or it might have a representat
 import * as resolve from 'table-resolver';
 
 // Or you can cherry-pick
-import { index } from 'table-resolver';
-import { index as resolveIndex } from 'table-resolver';
+import { nested } from 'table-resolver';
+import { nested as resolveNested } from 'table-resolver';
 ```
 
 ## API
```

Comment:
It would be a good idea to cover the new `_index` behavior here so you know which resolvers return it by default. Does this mean we can deprecate the old index resolver?